aws/amazon-chime-sdk-component-library-react
# Add Background Blur Provider and VideoInputBackgroundBlurControl component

**Issue #:** 

**Description of changes:**
Add the background blur feature to the React Library.

I created one provider - `BackgroundBlurProvider`and one sample component- `VideoInputBackgroundBlurControl`.

I modified `meetingManager.selectVideoInputDevice` to accept `VideoTransformDevice` along with `Device`.

There was also a small fix to initialize `selectedAudioInputTransformDevice` when the meeting is first started. We need to initialize that so that any subscribers get the correct initial value for `selectedAudioInputTransformDevice`
**Testing**
1. Have you successfully run `npm run build:release` locally?
Yes

2. How did you test these changes?
Locally ran tests
1. Join meeting, turn on video. 
2. Enable background blur
3. Disable BG Blur
4. Enable BG Blur and turn off video, turn on video
5. Switch between video inputs while BG BLur is on

3. If you made changes to the component library, have you provided corresponding documentation changes?
Yes, added storybook documentation for useBackgroundBlur, BackgroundBlurProvider, and VideoInputBackgroundBlurControl.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
